handbook/product/design/metrics/amplitude-for-designers.md

```diff
@@ -0,0 +1,89 @@
+# Amplitude for designers
+
+Amplitude provides session-based event data of Sourcegraph cloud features. This guide describes several key concepts and provides a tutorial on how designers can use Amplitude in the discovery process in addition to measuring results.
+
+## Key concepts
+
+### Events
+
+Events are recorded when a Sourcegraph EventLog action is triggered as defined in the product. Events include clicks against tracked elements, page views, and other interactions defined when building product features. All Sourcegraph events are ingested into Amplitude as soon as they are released.
+
+### Segmentation
+
+Segmentation is a way to limit the group of users you are analyzing to users matching with specific attributes. Users can be segmented by whether they are new or active, have cloud accounts, have added repositories, or if they belonged to a specific cohort (weekly group of users) or participated in an a/b test.
+
+Segments can help answer questions like “Are users from before or after feature X was released more likely to do Y?” and, “Are users with property A, more likely to perform event B more than 5 times a week?”
+
+### Notebooks
+
+Amplitude’s notebook feature is incredibly useful for telling the story behind data in a way charts and dashboards cannot. Designers can use these notebooks to collect and annotate charts that describe user behaviors, to report the results of their designs, and hypothesize about how to improve key metrics.
+
+### Dashboards
+
+Amplitude’s dashboards are a handy way to save related charts for later reference.
+
+---
+
+## Getting started with Amplitude
+
+Let’s start with asking the question, “How often do users view the sign-up page?“ and what events are correlated with this activity, then build a chart that tracks this data.
```

Comment:
```suggestion
Let’s start with asking the question, “How often do users view the sign-up page, and what events are correlated with this activity?" then build a chart that tracks this data.
```

Reply:
```suggestion
Let’s start with asking the following questions and building a chart that tracks this data. 
* How often do users view the sign-up page?
* What events are correlated with this activity? 
```

Reply:
@quinnkeast I see what you are pointing out, but I think we were both a bit wrong on the proper punctuation. For reference, with a single question in a sentence, here is the proper format. That said, I couldn't figure out formatting two so I restructured.